### ISSUE TITLE
Create an endpoint for saving/archiving poll results for future use

### ISSUE THREAD

**Issue by ntgomes** [ntgomes/PollCord#9]
POST endpoint → “/save”

Accepts request body in a format similar to this:
```
{
    “guild_id”: 1294898934,
    “poll_name”: “my_poll”,
    “results”: {
        “question1”: {
            "question_text": "How is the project going?",
            “Option1_text”: 10,
            “Option2_text”: 26,
            “Option3_text”: 13
        },
	“question2”...
    }
}
```

Returns the following response:
```
{
    "success": true/false
}
```

**Comment by AbhimanyuBellam** [comment]
I think we need to make a change here, from the above to:
```
{
    “guild_id”: 1294898934,
    “poll_name”: “my_poll”,
    “results”: {
        “question1”: {
            "question": "How is the project going?",
            “Option1”: 10,
            “Option2”: 26,
            “Option3”: 13
        },
	“question2”...
    }
}
```

**Comment by ntgomes** [comment]
Ah yes, good catch. Maybe call it question_text instead of question so that if there's already a question table or something... anything to save redundancy.

**Comment by AbhimanyuBellam** [comment]
Yes. Rewriting: 
```
{
    “guild_id”: 1294898934,
    “poll_name”: “my_poll”,
    “results”: {
        “question1”: {
            "question_text": "How is the project going?",
            “Option1”: 10,
            “Option2”: 26,
            “Option3”: 13
        },
	“question2”...
    }
}
```

**Comment by ntgomes** [comment]
Closed by #28
